src/components/v2/Layout/Footer/styles.ts

```diff
@@ -0,0 +1,73 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = () => {
+  const theme = useTheme();
+
+  const Container = css`
+    height: 56px;
+    padding: 0 ${theme.spacing(2)};
+    background-color: var(--color-bg-main);
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+
+    @media only screen and (max-width: 768px) {
+      height: auto;
+      padding: ${theme.spacing(3)} ${theme.spacing(2)};
+      flex-direction: column;
+    }
+  `;
+
+  const Status = css`
```

Comment:
In some places we write styles as an object here we are naming and exporting them directly with capital names. I think we should pick a style and stick to that style

Reply:
The outcome is the same: we return an object with named properties through a hook that returns a single object.

Reply:
I've updated the file to look like what you did on your PR for the header. I'm happy to follow this style from now 👍